src/components/MultiBarToggle.js

```diff
@@ -179,6 +184,13 @@ class MultiBarToggle extends Component {
                 pointerEvents="box-none"
                 style={toggleContainerStyle}
             >
+                {this.state.active && overlayActive ?
```

Comment:
Change ternary operator to AND
{this.state.active && overlayActive && (
  <JSX />
)}

Reply:
fixed!